packages/wallet/src/Delegation/computeImplicitCoin.ts

```diff
@@ -0,0 +1,31 @@
+import { ImplicitCoin } from '@cardano-sdk/cip2';
+import { ProtocolParametersRequiredByWallet } from '@cardano-sdk/core';
+import { sum } from 'lodash-es';
+import { InitializeTxProps } from '../types';
+
+/**
+ * Implementation is the same as in csl.get_implicit_input() and csl.get_deposit().
+ * Alternatively could build transaction body and use CSL to compute these.
```

Comment:
For now I'd prefer if we use this approach and just use the CSL for serialization.